fetchODBSchema.sh

```diff
@@ -1,5 +1,51 @@
 #!/bin/bash
 
+#
+# fetchODBSchema.sh
+#
+# Usage: sh fetchODBSchema.sh [local|staging]
+#
+# Fetches the current version of the ODB schema from either a 'local' ODB
+# instance running on localhost, or else from the 'staging' ODB.  Places it
+# in the proper place in `lucuma-schemas` for use in generating code.
+#
+
+KEY_MESSAGE=$(cat << END
```

Comment:
This is way more profession than what we had 😄  thanks

Reply:
Nice!

Reply:
Yeah. Very nice. Thanks for the improvements!